Makefile

```diff
@@ -426,8 +426,11 @@ terratest: # Run terratest suite
 		echo "$(RED)Make sure you run the tests against at least two running clusters$(NC)" ;\
 		exit 1;\
 	fi
-	cd terratest/test/ && go mod download && CLUSTERS_NUMBER=$(RUNNING_CLUSTERS) go test -v -timeout 15m -parallel=12 --tags=$(TEST_TAGS)
-
+	mkdir -p /tmp/terratest && cd terratest/test/ && go mod download && CLUSTERS_NUMBER=$(RUNNING_CLUSTERS) go test -v -timeout 15m -parallel=12 --tags=$(TEST_TAGS) | tee /tmp/terratest/all.log
+	curl --location --silent --fail --show-error -o terratest_log_parser https://github.com/gruntwork-io/terratest/releases/download/v0.38.8/terratest_log_parser_linux_amd64
+	chmod +x terratest_log_parser
+	sudo mv terratest_log_parser /usr/local/bin
+	terratest_log_parser --testlog /tmp/terratest/all.log --outputdir /tmp/terratest
```

Comment:
It's not very obvious, but 
.github/workflows/terratest-more-clusters.yaml
currently runs only the [full_round_robin](https://github.com/k8gb-io/k8gb/blob/master/terratest/test/k8gb_full_roundrobin_test.go) tests. So there is no log interleaving coming from this output.

it would make more sense to have it for
.github/workflows/terratest.yaml

.. which combines multiple tests together and the output is mixed

Btw. the logic that controls when certain tests are run is done using build tags